styles/badge.css

```diff
@@ -0,0 +1,62 @@
+.badge {
+    border: 1px solid transparent;
```

Comment:
is there any use to have a transparent border? Am I missing something?

Reply:
Since I am using box-sizing, I have included 1px of border. Mostly so that when the border color is changed on hover, it does not distort the whole element.
Nevertheless, on badge I think this property could be removed. Will check this out. Thanks